Header header
rasoul_visualizer_pkg/GLcmd[] cmd

# OpenGL commands

uint16 glEnd=0

uint16 glBegin_GL_POINTS=1
uint16 glBegin_GL_LINES=2
uint16 glBegin_GL_LINE_STRIP=3
uint16 glBegin_GL_LINE_LOOP=4
uint16 glBegin_GL_TRIANGLES=5
uint16 glBegin_GL_TRIANGLE_STRIP=6
uint16 glBegin_GL_TRIANGLE_FAN=7
uint16 glBegin_GL_QUADS=8
uint16 glBegin_GL_QUAD_STRIP=9
uint16 glBegin_GL_POLYGON=10
uint16 glVertex3f=11
uint16 glColor3f=12
uint16 glColor4f=13
uint16 glNormal3f=14
uint16 glPushMatrix=15
uint16 glPopMatrix=16
uint16 glMultMatrixf=17
uint16 glLineWidth=18
uint16 glPointSize=19
uint16 writeText=20
uint16 glLoadIdentity=21
uint16 glutSolidCone=22
uint16 glTranslatef=23
uint16 glRotatef=24
uint16 glTexturize=25
uint16 glTexCoord2f=26
uint16 glDisable_GL_TEXTURE_2D=27
uint16 glEnable_GL_TEXTURE_2D=28
uint16 drawSolidCylinder=29
uint16 drawWiredCylinder=30
uint16 writeText2=31
uint16 writeText3=32
